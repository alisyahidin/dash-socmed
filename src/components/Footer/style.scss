.footer {
  color: #999;
  &__link {
    color: #999 !important;
    &:hover {
      text-decoration: none;
      color: #666 !important;
    }
  }
}