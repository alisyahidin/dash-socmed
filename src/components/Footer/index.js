import React from 'react'
import {
  FaGithub,
  FaLinkedin,
  FaFacebook
} from 'react-icons/fa'
import './style.scss'

export default function Footer(props) {
  return (
    <div className="footer">
      <p>Find me:</p>
      <p className="d-flex">
        <a href="https://github.com/alisyahidin" className="footer__link mr-2 d-flex align-items-center">
          <FaGithub className="mr-1" />alisyahidin
        </a>
        <a href="https://linkedin.com/in/alisyahidin" className="footer__link mr-2 d-flex align-items-center">
          <FaLinkedin className="mr-1" />alisyahidin
        </a>
        <a href="https://facebook.com/Alisyahidin19" className="footer__link mr-2 d-flex align-items-center">
          <FaFacebook className="mr-1" />alisyahidin19
        </a>
      </p>
      <p className="mt-2">Coding with Love at Kuningan. &copy; 2019</p>
    </div>
  )
}