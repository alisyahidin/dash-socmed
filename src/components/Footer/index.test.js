import React from 'react'
import { shallow, mount } from 'enzyme'
import Footer from './index'

it('render <Footer /> correctly', () => {
  const component = shallow(<Footer />)

  expect(component).toMatchSnapshot()
})