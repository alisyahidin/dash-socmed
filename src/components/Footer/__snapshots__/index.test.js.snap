// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <Footer /> correctly 1`] = `
<div
  className="footer"
>
  <p>
    Find me:
  </p>
  <p
    className="d-flex"
  >
    <a
      className="footer__link mr-2 d-flex align-items-center"
      href="https://github.com/alisyahidin"
    >
      <FaGithub
        className="mr-1"
      />
      alisyahidin
    </a>
    <a
      className="footer__link mr-2 d-flex align-items-center"
      href="https://linkedin.com/in/alisyahidin"
    >
      <FaLinkedin
        className="mr-1"
      />
      alisyahidin
    </a>
    <a
      className="footer__link mr-2 d-flex align-items-center"
      href="https://facebook.com/Alisyahidin19"
    >
      <FaFacebook
        className="mr-1"
      />
      alisyahidin19
    </a>
  </p>
  <p
    className="mt-2"
  >
    Coding with Love at Kuningan. Â© 2019
  </p>
</div>
`;
